src/formatter.js

```diff
@@ -52,21 +52,29 @@ module.exports = function(results) {
     output += '##teamcity[testStarted name=\'' + reportName + ': ' +
                escapeTeamCityString(result.filePath) + '\']\n';
 
-    var messageList = [];
+    var errorsList = [];
+    var warningsList = [];
+    var userMessage;
 
     messages.forEach(function(message) {
-      if (message.severity === 2) {
-        messageList.push(
-          'line ' + message.line +
-          ', col ' + message.column + ', ' + message.message
-        );
+      userMessage = 'line ' + (message.line || 0) +
```

Comment:
no need to declare userMessage in the outer scope and mutating it on every loop.

rather just `var userMessage = ...`